File: block4/task6.py
a = float(input("Введите первое число: "))
b = float(input("Введите второе число: "))


if a > b:
    print("Больше:", a)
elif b > a:
    print("Больше:", b)
else:
    print("Числа равны")


if a < b:
    print("Меньше:", a)
elif b < a:
    print("Меньше:", b)
else:
    print("Числа равны")

File: web41/js5dz/task6.js
const prompt = require('prompt-sync')();

function fibonacci(n) {
    if (n <= 0) {
        return 0;
    }
    if (n === 1 || n === 2) {
        return 1;
    }
    return fibonacci(n - 1) + fibonacci(n - 2);
}

const input = prompt('Введите порядковый номер числа Фибоначчи: ');

if (isNaN(input) || input.trim() === '') {
    console.log('Пожалуйста, введите корректное число.');
} else {
    const number = parseInt(input, 10);
    const result = fibonacci(number);
    console.log(`Число Фибоначчи с порядковым номером ${number}: ${result}`);
}
